pkg/telemetry/histogram.go

```diff
@@ -18,6 +18,8 @@ var (
 		1_500, 2_000, 4_000,
 		8_000, 15_000,
 	}
+
+	peekSizeBoundaries = []float64{10, 30, 50, 100, 250, 500, 1000}
```

Comment:
I suspect we'll end up also allowing a peek up to 2.5 and 5k shortly, if that's possible to add?  I'd like to make peeks dynamic based off of account heuristics and queue size at some point

Reply:
Will add